src/patternfly/components/Nav/nav.scss

```diff
@@ -4,181 +4,96 @@
   --pf-c-nav--Width: #{pf-size-prem(290px)};
   --pf-c-nav--Transition: var(--pf-global--Transition);
 
-  // List link
+  // Nav List link
   --pf-c-nav__list-link--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--md--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-nav__list-link--PaddingX: var(--pf-global--spacer--md);
   --pf-c-nav__list-link--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-nav__list-link--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-nav__list-link--md--PaddingLeft: var(--pf-global--spacer--xl);
-
-  // font weight
+  --pf-c-nav__list-link--md--PaddingX: var(--pf-global--spacer--xl);
   --pf-c-nav__list-link--FontWeight: var(--pf-global--FontWeight--normal);
-  --pf-c-nav__list-link--active--FontWeight: var(--pf-global--FontWeight--normal);
-  --pf-c-nav__list-link--focus--FontWeight: var(--pf-global--FontWeight--normal);
-
-  // color
   --pf-c-nav__list-link--Color: var(--pf-global--Color--dark-100);
   --pf-c-nav__list-link--hover--Color: var(--pf-global--link--Color);
-  --pf-c-nav__list-link--active--Color: var(--pf-global--link--Color);
-  --pf-c-nav__list-link--focus--Color: var(--pf-global--link--Color);
-
-  // background color
   --pf-c-nav__list-link--BackgroundColor: transparent;
   --pf-c-nav__list-link--hover--BackgroundColor: transparent;
-  --pf-c-nav__list-link--active--BackgroundColor: transparent;
-  --pf-c-nav__list-link--focus--BackgroundColor: transparent;
-
-  // after
-  --pf-c-nav__list-link--after--Left: var(--pf-c-nav__list-link--PaddingLeft);
-  --pf-c-nav__list-link--after--md--Left: var(--pf-c-nav__list-link--md--PaddingLeft);
   --pf-c-nav__list-link--after--BackgroundColor: transparent;
   --pf-c-nav__list-link--hover--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__list-link--active--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__list-link--focus--after--BackgroundColor: var(--pf-global--link--Color);
+  --pf-c-nav__list-link--after--Left: var(--pf-c-nav__list-link--PaddingX);
+  --pf-c-nav__list-link--after--md--Left: var(--pf-c-nav__list-link--md--PaddingX);
+  --pf-c-nav__list-link--after--Width: 3.125rem; // hardcoding border bottom to match design spec
+  --pf-c-nav__list-link--after--Height: 3px; // hardcoding border bottom to match design spec
```

Comment:
Can we use `pf-size-prem` function to create width and height values?

Reply:
updated